### PR TITLE
Webpack4 migrate

### PR DESCRIPTION
**Pull request by againksy**
this is the migration to webpack 4 and react 16

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-webpack-node",
-  "version": "3.4.1",
+  "version": "4.0.0",
```
[the remark this answers is not in the document]
Comment (againksy) [reply package.json]: @GGAlanSmithee  maybe 3.5.0 or some funny 3.4.4444 ?

File: webpack/paths.js [hunk 2]
```diff
@@ -12,6 +12,7 @@ module.exports = {
   assets: path.resolve(CURRENT_WORKING_DIR, 'public', 'assets'),
   compiled: path.resolve(CURRENT_WORKING_DIR, 'compiled'),
   public: '/assets/', // use absolute path for css-loader?
+  prod_public: '/', // use absolute path for css-loader?
```
[the remark this answers is not in the document]
Comment (againksy) [reply webpack/paths.js]: @GGAlanSmithee in prod with  `public: '/assets/'` the app tries to get the js and css from  '/assets//assets/*'  i did not searched why- just changed the prod publicPath for '/'.
I ll add comments about this.

File: package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (againksy) [reply package.json]: @GGAlanSmithee what you think? I ll set  it

**Comment by GGAlanSmithee** [comment]
@againksy this is great - thanks a million!

**Pull request merged**